<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>habits_module</class>
 <widget class="QWidget" name="habits_module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: white;
	margin-left: 20px;
	margin-top: 0;
	margin-right: 20px;
	border-radius: 10px;
	padding: 15px;
	/*padding: 10px;*/
	/*margin: 60px 20px 80px;*/
}

QLabel {
  color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_habit_name">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 20px;
margin-bottom: 0;
margin-top: 0;</string>
     </property>
     <property name="text">
      <string>Titulo h치bito</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="styleSheet">
      <string notr="true">color: lightgray;
background-color: lightgray;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="description_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
  padding: 0;
  margin-top: 0;
  margin-bottom: 0;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_desc_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
  font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Descripci칩n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_habit_desc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	padding: 5px;
	border: 0px solid gray;
}</string>
        </property>
        <property name="text">
         <string>Descripci칩n</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="frequency_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
  padding: 0;
  margin-top: 0;
  margin-bottom: 0;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_freq_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
  font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Frecuencia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">max-width: 35px;
padding-left: 0;
margin-right: 0;
margin-left: 10px;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>游늰</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_habit_freq">
           <property name="styleSheet">
            <string notr="true">QLabel {
	padding: 0px;
	border: 0px solid gray;
	margin-left: 0px;
}</string>
           </property>
           <property name="text">
            <string>Frecuencia</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons_widget" native="true">
     <property name="styleSheet">
      <string notr="true">margin: 20px;
font-weight: bold;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_edit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    padding: 10px 20px;
    border-radius: 6px;
    border: none;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #0056b3;
}

QPushButton:pressed {
    background-color: #004085;
}</string>
        </property>
        <property name="text">
         <string>Editar h치bito</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: red;
    color: white;
    padding: 10px 20px;
    border-radius: 6px;
    border: none;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #d32f2f;
}

QPushButton:pressed {
    background-color: #b71c1c;
}</string>
        </property>
        <property name="text">
         <string>Eliminar h치bito</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpaces">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">color: lightgray;
background-color: lightgray;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">margin-top: 0;
margin-bottom: 0;

max-height: 25px;
color: gray;</string>
     </property>
     <property name="text">
      <string>Aqu칤 puedes ver el detalle de un h치bito, puedes editarlo o eliminarlo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
